CONTRIBUTING.rst

```diff
@@ -49,34 +45,92 @@ remember that SimPEG is maintained on a volunteer basis. If you suggest an
 enhancement, we certainly appreciate if you are also willing to take action
 and start a pull request!
 
+.. _questions:
+
+Questions
+---------
+
+If you have a question regarding a specific use of SimPEG, the fastest way
+to get a response is by posting on our Discourse discussion forum:
+https://simpeg.discourse.group/. Alternatively, if you prefer real-time chat,
+you can join our slack group at http://slack.simpeg.xyz.
+Please do not create an issue to ask a question.
+
```

Comment:
Also, should we include a heading on meetings? This is a great way for folks who are interested in contributing to engage with us.

Reply:
I think I'll add it in README.txt so it's a little more front and center.